src/bdev_async.rs

```diff
@@ -29,6 +30,14 @@ pub struct BdevAsyncCallContext {
     pub arg: *mut c_void,
 }
 
+/// Defines mode for Bdev IOStats reset.
+pub enum BdevResetMode {
+    /// Resets all IoStat counters for a given Bdev.
+    SpdkBdevResetStatAll = 0,
```

Comment:
Can't we pull these from libspdk?

Reply:
Done, We could import the enum variant and use.